tests/test_models/test_editors/test_nafnet/test_baseline.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
file name should be the same with source file

Reply:
Done.